src/less-style/tree-content.less

```diff
@@ -89,10 +89,6 @@
   }
 }
 
-#jstree-marker {
```

Comment:
Didn't find any references to this id. Thought it might be the markup that appears when you drag something from the tree, but it isn't.